distribution/docs/src/main/jdocs/content/_configuring/managing-apps-from-command-console-contents.adoc

```diff
@@ -0,0 +1,37 @@
+:title: Managing Applications From the ${command-console}
+:type: configuringCommandConsole
+:status: published
+:summary: Managing Applications From the ${command-console}.
```

Comment:
Some of the summaries, especially those with the same text as the title have words starting in (mostly) caps, but others only have the first word in caps, like a normal sentence.